@here/harp-omv-datasource/lib/OmvDecodedTileEmitter.ts

```diff
@@ -259,7 +256,18 @@ export class OmvDecodedTileEmitter implements IOmvEmitter {
 
                 // Always store the position, otherwise the following POIs will be
                 // misplaced.
-                positions.push(x, y, z);
+                const worldPos = tile2world(
```

Comment:
this piece of code looks like it should be a (local) function)

Reply:
Absolutely right, will do.